### PR TITLE
Use "inline" source maps in the compiler (part of #1375)

### PR DESCRIPTION
**Pull request by georgiy-abbasov**

### CODE REVIEW

File: src/compiler/es-next/index.js [hunk 1]
```diff
@@ -87,14 +87,9 @@ export default class ESNextCompiler {
...
+            hookRequire:              true,
             handleUncaughtExceptions: false,
-            environment:              'node',
-
-            retrieveSourceMap: filename => {
-                var map = this.sourceMaps[filename];
-
-                return map ? { url: filename, map } : null;
-            }
+            environment:              'node'
```
Comment (inikulin) [inline src/compiler/es-next/index.js]: But how we'll specify sourcemaps for `sourceMapSupport` then?

Comment (georgiy-abbasov) [inline src/compiler/es-next/index.js]: They will obtain automatically, ` retrieveSourceMap:` argument used to be custom implementation of this process instead `source-maps-support` default.

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
It [works](https://www.screencast.com/t/vMqH5fdlcLEO) in chrome-dev tools on Ubuntu 16.04

**Comment by georgiy-abbasov** [comment]
Also it works on OS X with node js 7.8.0, chrome dev 59 and chrome canary 59
https://screencast.com/t/xOZc7KWEL

**Comment by georgiy-abbasov** [comment]
Comparsion
Previous approach
https://www.screencast.com/t/sYt1uzw9F
https://www.screencast.com/t/BTG4SKdle

Current approach
https://www.screencast.com/t/qaqhp0eWq4
https://www.screencast.com/t/YFKdf7pF

**Comment by AlexanderMoskovkin** [comment]
rebase required

**Comment by georgiy-abbasov** [comment]
FPR

**Pull request merged**